lib/outputs/outputs_test.py

```diff
@@ -108,8 +108,9 @@ def test_fill_file_descriptors(self):
                 remove(expected[index][output_mode].name)
 
     def test_set_report_color(self):
+        orange = "\033[38;5;208m"
```

Comment:
Config file please, you are setting this value multiple times across several files.

Reply:
Done